docs/content.zh/release-notes/flink-1.13.md

```diff
@@ -0,0 +1,227 @@
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License.
+-->
+
+# Release Notes - Flink 1.13
+
+These release notes discuss important aspects, such as configuration, behavior, or dependencies,
+that changed between Flink 1.12 and Flink 1.13. Please read these notes carefully if you are
+planning to upgrade your Flink version to 1.13.
+
+### Failover
+
+#### Remove state.backend.async option.
+
+##### [FLINK-21935](https://issues.apache.org/jira/browse/FLINK-21935)
+
+The `state.backend.async` option is deprecated. Snapshots are always asynchronous now (as they were
+by default before) and there is no option to configure a synchronous snapshot any more.
+
+The constructors of `FsStateBackend` and `MemoryStateBackend` that take a flag for sync/async
+snapshots are kept for API compatibility, but the flags are ignored now.
+
+#### Unify binary format for Keyed State savepoints
+
+##### [FLINK-20976](https://issues.apache.org/jira/browse/FLINK-20976)
+
+Flink’s savepoint binary format is unified across all state backends. That means you can take a
+savepoint with one state backend and then restore it using another.
+
+If you want to switch the state backend you should first upgrade your Flink version to 1.13, then
+take a savepoint with the new version, and only after that, you can restore it with a different
+state backend.
+
+#### FailureRateRestartBackoffTimeStrategy allows one less restart than configured
+
+##### [FLINK-20752](https://issues.apache.org/jira/browse/FLINK-20752)
+
+The Failure Rate Restart Strategy was allowing 1 less restart per interval than configured. Users
+wishing to keep the current behavior should reduce the maximum number of allowed failures per
+interval by 1.
+
+#### Support rescaling for Unaligned Checkpoints
+
+##### [FLINK-17979](https://issues.apache.org/jira/browse/FLINK-17979)
+
+While recovering from unaligned checkpoints, users can now change the parallelism of the job. This
+change allows users to quickly upscale the job under backpressure.
+
+### SQL
+
+#### Officially deprecate the legacy planner
+
+##### [FLINK-21709](https://issues.apache.org/jira/browse/FLINK-21709)
+
+The old planner of the Table & SQL API is deprecated and will be dropped in Flink 1.14. This means
+that both the BatchTableEnvironment and DataSet API interop are reaching end of life. Use the
+unified TableEnvironment for batch and stream processing with the new planner, or the DataStream API
+in batch execution mode.
+
+#### Disable problematic cast conversion between NUMERIC type and TIMESTAMP type
+
+##### [FLINK-21698](https://issues.apache.org/jira/browse/FLINK-21698)
+
+The CAST operation between `NUMERIC` type and `TIMESTAMP` type is problematic and is disabled now,
+e.g.
+`CAST(numeric AS TIMESTAMP(3))` is disabled and should use `TO_TIMESTAMP(FROM_UNIXTIME(numeric))`
+instead.
```

Comment:
```suggestion
#### Use TIMESTAMP_LTZ as return type for function PROCTIME()

#### [FLINK-21714](https://issues.apache.org/jira/browse/FLINK-21714)

Before Flink 1.13, the function return type of `PROCTIME()` is `TIMESTAMP`, and the return value is the `TIMESTAMP` in UTC time zone,
e.g. the wall-clock shows `2021-03-01 12:00:00` at Shanghai, however the `PROCTIME()` displays `2021-03-01 04:00:00` which is wrong.
Flink 1.13 fixes this issue and uses `TIMESTAMP_LTZ` type as return type of `PROCTIME()`, users don't need to deal time zone problems anymore.

#### Support defining event time attribute on TIMESTAMP_LTZ column

#### [FLINK-20387](https://issues.apache.org/jira/browse/FLINK-20387)

Support defining event time attribute on TIMESTAMP_LTZ column, base on this, Flink SQL gracefully support the Daylight Saving Time. 

#### Correct function CURRENT_TIMESTAMP/CURRENT_TIME/CURRENT_DATE/LOCALTIME/LOCALTIMESTAMP/NOW()

##### [FLINK-21713](https://issues.apache.org/jira/browse/FLINK-21713)

The value of time function CURRENT_TIMESTAMP and NOW() are corrected from UTC time with `TIMESTAMP` type to epoch time with `TIMESTAMP_LTZ` type. Time function LOCALTIME, LOCALTIMESTAMP, CURRENT_DATE, CURRENT_TIME, CURRENT_TIMESTAMP and NOW() are corrected from evaluates for per record in batch mode to evaluate once at query-start for batch job.

#### Disable problematic cast conversion between NUMERIC type and TIMESTAMP type

##### [FLINK-21698](https://issues.apache.org/jira/browse/FLINK-21698)

The CAST operation between `NUMERIC` type and `TIMESTAMP` type is problematic and is disabled now,
e.g.
`CAST(numeric AS TIMESTAMP(3))` is disabled and should use `TO_TIMESTAMP(FROM_UNIXTIME(numeric))`
instead.
```

would be better if we can add these incompatibility note here, @dawidwys

Reply:
We will add them here, if you add them to the corresponding issue.